adoc/chapters/programming_interface.adoc

```diff
@@ -6475,10 +6487,12 @@ following values for the [code]#AccessMode# template parameter:
     | Access a buffer from a <<host-task>>.
 |====
 
-Programs which specify the access target as [code]#target::device# and then use
-the [code]#accessor# from a <<host-task>> are ill formed.  Likewise, programs
-which specify the access target as [code]#target::host_task# and then use the
-[code]#accessor# from a <<sycl-kernel-function>> are ill formed.
+Programs which specify the access target as [code]#target::device# and then
+capture the [code]#accessor# in a <<host-task>> can only use the accessor for
+interoperability through the [code]#interop_handle#.
```

Comment:
```suggestion
capture the [code]#accessor# in a <<host-task>> can use the accessor for
interoperability through the [code]#interop_handle#.  Any other use of of such an
accessor in a <<host-task>> results in undefined behavior.
```

I think we should state explicitly that you get UB if you use a `device` accessor in any other way from a host task.

Please also close internal issue 609 when this is merged.

Reply:
Updated as suggested to specify that it is undefined behavior.